app/models/bike.rb

```diff
@@ -505,6 +505,20 @@ def registration_address # Goes along with organization additional_registration_
     end
   end
 
+  def registration_location
+    address = registration_address.with_indifferent_access
+    city = address[:city]&.titleize
+    state = address[:state]&.upcase
+
+    if city && state
+      "#{city}, #{state}"
```

Comment:
Nit: you could just do `[city, state].reject(&:blank?).join(", ")`

Reply:
If city is blank that'll produce `", NY"`, e.g.

Reply:
Thanks! Updated in f6e18823